modules/firehose/config.go

```diff
@@ -33,6 +35,7 @@ var (
 type moduleConfig struct {
 	State        string                 `json:"state"`
 	ChartVersion string                 `json:"chart_version"`
+	StopByTime   *time.Time             `json:"stop_by_time"`
```

Comment:
i know this is draft but just leaving this as early feedback. 

This name is bit misleading. because "stop-by-time" sounds like it will stop the firehose before the set time. But the actual behaviour is "stop exactly-at or as soon as possible after set time"

Reply:
Cool, going ahead with `StopTime`.